source/Irrlicht/COpenGLExtensionHandler.cpp

```diff
@@ -1026,6 +1028,19 @@ void COpenGLExtensionHandler::loadFunctions()
         InvocationSubGroupSize[1] = 32;
     }
 
+    if (FeatureAvailable[IRR_ARB_spirv_extensions])
+    {
+        glGetIntegerv(GL_NUM_SPIR_V_EXTENSIONS, reinterpret_cast<GLint*>(&SPIR_VextensionsCount));
```

Comment:
I think those are only supported if you have GL 4.6 and can feed spir-v directly to OpenGL

>  This extension requires ARB_gl_spirv.